drivers/serial/uart_pl011.c

```diff
@@ -748,6 +748,7 @@ static void pl011_txint(FAR struct uart_dev_s *dev, bool enable)
   if (enable)
     {
       pl011_irq_tx_enable(sport);
+      spin_unlock_irqrestore(&sport->lock, flags);
 
       /* Fake a TX interrupt here by just calling uart_xmitchars() with
        * interrupts disabled (note this may recurse).
```

Comment:
doesn't spin_unlock_irqrestore enable interrupts?

Reply:
do you mean this comment ("with interrupts disabled") is referring to the critical section?
well, it makes sense, but quite confusing.

Reply:
let's remove the comment to avoid he confusing.

Reply:
done